docker/dev.sh

```diff
@@ -151,3 +151,11 @@ docker run -itd \
   -v $(pwd):/opt/openoni:Z \
   -v $(pwd)/docker/data:/opt/openoni/data:z \
   open-oni:dev
+
+port=$(_port $APP_URL)
```

Comment:
From both looking at the code and testing this with `DOCKERPORT=8080`, I don't see how this will ever match. `$APP_URL` never seems to have the port added to it to be pulled out by the `_port` function.